package.json

```diff
@@ -26,17 +26,24 @@
 		"test-watch": "jest --config jest/jest.config.js --watch",
 		"lint": "eslint '*/**/*.{js,jsx,ts,tsx}' --quiet --fix",
 		"storybook": "start-storybook -p 6006 -c .storybook",
-		"prepare": "(test -d ./.git && npx husky install) || true"
+		"prepare": "(test -d ./.git && npx husky install) || true",
+		"build-fabric": "test -d ./node_modules/fabric && cd ./node_modules/fabric && node build.js modules=ALL exclude=assessors || true",
```

Comment:
fabric.js is a canvas library, it creates canvas interactions like drawing with brush, pinch to zoom and panning. the package comes with optional modules and requires a custom build to include those modules. the `build-fabric` script does that and is included as part of `postinstall` step.